test/remote/gateways/remote_element_test.rb

```diff
@@ -159,4 +159,13 @@ def test_transcript_scrubbing_with_echeck
     assert_scrubbed(@check.routing_number, transcript)
     assert_scrubbed(@gateway.options[:account_token], transcript)
   end
+
+  def test_successful_purchase_with_card_present_code
```

Comment:
It looks like this method might more logically be located and modeled after `test_successful_purchase_with_shipping_address`.

Reply:
https://github.com/activemerchant/active_merchant/pull/3623/commits/548f0082dd3b100808794f0895231bcfb804d56f#

Moved the test case immediately after `test_successful_purchase_with_shipping_address`. Also modeled after that method.